components/resources/brave_components_resources.grd

```diff
@@ -36,15 +36,19 @@
       <include name="IDR_BRAVE_NEW_TAB_IMG2" file="../img/newtab/settings_prefs_btn.svg" type="BINDATA" />
 
       <!-- WebUI wallet resources -->
+      <include name="IDR_WALLET_PAGE_HTML" file="../brave_wallet_ui/page/wallet_page.html" type="BINDATA" />
```

Comment:
do we need to include the whole wallet page for Android?

Reply:
fixed